modules/machine-config-pin-preload-images.adoc

```diff
@@ -0,0 +1,141 @@
+// Module included in the following assemblies:
+//
+// * machine_configuration/machine-config-pin-preload-images-about.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="machine-config-pin-preload-images_{context}"]
+= Pinning and preloading images
+
+You can pin and preload images by using a `PinnedImageSet` custom resource (CR). The pinned image set defines the list of images to preload and the machine config pool to which the images should be pinned. 
+
+The images are stored in the the `/etc/crio/crio.conf.d/50-pinned-images` file on the nodes. 
+
+.Procedure
+
+. Create a YAML file that defines the `PinnedImageSet` object, similar to the following example:
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: PinnedImageSet
+metadata:
+  labels: <1>
+    machineconfiguration.openshift.io/role: worker
+  name: worker-pinned-images
+spec:
+  pinnedImages: <2>
+   - name: quay.io/openshift-release-dev/ocp-release@sha256:513cf1028aa1a021fa73d0601427a0fbcf6d212b88aaf9d76d4e4841a061e44e
+   - name: quay.io/openshift-release-dev/ocp-release@sha256:61eae2d261e54d1b8a0e05f6b5326228b00468364563745eed88460af04f909b
+----
++
```

Comment:
Just curious -- is there a reason why you use one style of callout on lines 31 - 36, but a different style of callout elsewhere in this module (for example, see line 75)?

Reply:
I was/am confused over whether we are supposed to move away from <1> style callouts. Couldn't decide apparently! I will make them both `where` type.